packages/panzer/LICENSE

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
why doesn't the LICENSE file note that it is BSD-3?

Reply:
Yeah, I wondered that myself, but this is what is stated by [SPDX](https://spdx.org/licenses/BSD-3-Clause.html).

Reply:
I guess I would break with SPDX here and add a single line with the license type. As a developer, I download a package and look at the license file. Right now this is user hostile - just a bunch of paragraphs of lawyer speak. If it says BSD-clause-3 then it's a two second thing that users all understand instead of trying to guess the license.